src/prompt/files_prompt.rs

```diff
@@ -42,6 +42,11 @@ impl<'a> FilesPrompt<'a> {
             .get_figlet()
             .expect("Ensure figlet_file points to a valid file, or remove it.");
 
+        // Padded limit (never overflows by 1 item)
+        let total = self.options.len();
+        let max = 15;
+        let take = if total > max { max - 2 } else { total + 1 };
```

Comment:
It seems strange that it jumps back down after passing the threshold. Is this intended?

- total = 14, then take = 15
- total = 15, then take = **16**
- total = 16, then take = **13**
- total = 17, then take = 13

Reply:
That's what was already happening? I only made slight edits to the values. I can take a look at that though.

I'd like to swap out the static max to depend on the terminal height as well at some point.